datastruc_ex7.py

```diff
@@ -0,0 +1,79 @@
+import random as rnd
+
+# ex7
+# a
+
+
+def func(n: int, random: bool, lst: list):
+    with open('WordData.txt', 'r') as file:
+        allText = file.read()
+        if random:
+            text_list = allText.split('\n')
+            randomlist = rnd.sample(range(0, len(text_list)), n)
+            selected_words = []
+            for i in range(len(randomlist)):
+                selected_words.append(text_list[randomlist[i]])
+            return selected_words
+        else:
+            text_list = allText.split('\n')
+            selected_words = []
+            for i in range(len(lst)):
+                selected_words.append(text_list[lst[i]])
+            return selected_words
+
+
+# b
+
+def freqDict(s: str):
+    '''
+    A function which creates a frequency dictionary from a string
+    '''
+    all_freq = {}
+    for i in s:
+        if i in all_freq:
+            all_freq[i] += 1
+        else:
+            all_freq[i] = 1
+    return all_freq
+
+
+def freqFunc(lst: list):
```

Comment:
Bad name, does not relate to the functions functionality

Reply:
I agree. already changed to more informative one